<launch>
 <!-- joy node -->
  <node respawn="true" pkg="joy" type="joy_node" name="turtle_joy" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <!--重复发送手柄话题-->
    <param name="autorepeat_rate" type="double" value="10" />
    <param name="deadzone" value="0.12" />
  </node>

 <!-- Axes -->
  <param name="axis_linear_lx" value="1" type="int"/>
  <param name="axis_linear_rx" value="3" type="int"/>
  <param name="axis_linear_y" value="4" type="int"/>
  <param name="axis_angular_lz" value="0" type="int"/>
  <param name="axis_angular_rz" value="2" type="int"/>

  <param name="scale_linear_lx" value="0.3" type="double"/>
  <param name="scale_linear_rx" value="0.5" type="double"/>
  <param name="scale_linear_y" value="0.3" type="double"/>
  <param name="scale_angular_lz" value="0.3" type="double"/>
  <param name="scale_angular_rz" value="0.5" type="double"/>
  <node pkg="ros_joy" type="teleop_joy" name="teleop"/>

  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find surfel_fusion)/launch/surfel_map.rviz"/>
</launch>
